crates/ruff_python_formatter/src/other/keyword.rs

```diff
@@ -13,10 +13,11 @@ impl FormatNodeRule<Keyword> for FormatKeyword {
             arg,
             value,
         } = item;
-        if let Some(argument) = arg {
-            write!(f, [argument.format(), text("=")])?;
+        if let Some(arg) = arg {
```

Comment:
Nit: I prefer to move the logic that is common for all branches out of the `if`. I find that this improves readability. I otherwise have to manually parse both branches to understand what's the same/different.

Reply:
i want to keep those two specifically since they are one entry in a list each and because each branch has its own comment handling